src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -410,13 +418,28 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
             // Set limit to the query for pagination support
             metadataQueryClone.limit = DEFAULT_PAGE_SIZE;
         }
+
+        if (sortBy && sortDirection) {
```

Comment:
there are default props so wouldn't this `if` always be true? which means the customers sort in query is always overwritten?